### DISCUSSION TITLE
Lambda Asset bundling follow mode

### DISCUSSION THREAD

**Discussion by phuctm97** [aws/aws-cdk#13431] [Q&A]
Hi AWS CDK team,

I'm bundling asset for my lambda which includes some symbolic links that I want to keep (not follow) in output zip file, how can I do that?

Try to look at the docs, but it said deprecated.

Thank you!

**Comment by tim-finnigan** [comment]
Hi @phuctm97, checking in as this discussion was opened a while ago. Which docs were you looking at that said deprecated? Here is the documentation for `SymlinkFollowMode`: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.SymlinkFollowMode.html.

However — there is an open bug stating that symlinks are always followed on publish and FollowMode has no effect: https://github.com/aws/aws-cdk/issues/9251
